udata/core/dataset/croquemort.py

```diff
@@ -64,8 +64,8 @@ def check_url_from_cache(url, group=None):
         response = requests.get(retrieve_url,
                                 params={'url': url, 'group': group},
                                 timeout=TIMEOUT)
-    except requests.ConnectionError:
-        return {'error': 'URL checker not found'}, {}
+    except requests.RequestException:
```

Comment:
The "not found" case is tested on the line below (`response.status_code == 404`).
As the `timeout` parameter is given as parameter, `TimeoutException` can be raised.
Am I wrong ?